pom.xml

```diff
@@ -47,7 +47,6 @@
     <build>
         <sourceDirectory>src/main/scala</sourceDirectory>
         <scriptSourceDirectory>src/main/scripts</scriptSourceDirectory>
-        <testSourceDirectory>src/test/scala</testSourceDirectory>
```

Comment:
The ScalaTest compiles tests from the src/test/scala path by default.
We compile tests from both paths: src/test/scala, src/test/java.
Please look at build.log file - https://travis-ci.org/swagger-api/swagger-core/builds/68503343#L3622-L3623